scss/components/_tabs.scss

```diff
@@ -52,11 +52,6 @@
     display: inline-block;
   }
 
-  //hide horizontal scrollbars
-  .scrollbar-container.horizontal {
```

Comment:
I think this is for the tabs on the CC menu, do we want to show the scrollbar here as well? The tabs have left and right arrows currently.

Reply:
@jpfranco yeah I thought that was affecting the quality page too but it wasn't actually, reverted it